Edits to Kathryn's OSS champions blog (#1836)
OpenLiberty/blogs: posts/2021-10-19-KathrynKodama_MeetTheTeam.adoc

Open Liberty Dev Mode is a hot deployment tool that is made available as a goal of the Liberty Maven plug-in or as a task of the Liberty Gradle plug-in. This enables developers to iterate quickly when developing their cloud-native Java applications with Open Liberty.

The link:https://github.com/eclipse/lsp4mp[Language Server for MicroProfile (Eclipse LSP4MP)] and link:https://github.com/eclipse/lsp4jakarta[Language Server for Jakarta EE (Eclipse LSP4Jakarta)] both offer language specific support in an IDE or editor for their respective technologies. They help to deliver language support capabilities, such as diagnostics, code completes, hover for more information, directly within the developer's IDE of choice. 

The link:https://github.com/MicroShed/mp-starter-vscode-ext[MicroProfile Starter for VS Code] which makes use of the MicroProfile Starter (https://start.microprofile.io/) project, the VS Code Extension Pack for MicroProfile, and the Open Liberty Tools for VS Code and IntelliJ projects are all IDE specific plugins that users can download from the VS Code or IntelliJ marketplace.

> The Open Liberty Tools extensions for link:https://github.com/OpenLiberty/open-liberty-tools-vscode[VS Code] and link:https://github.com/OpenLiberty/open-liberty-tools-intellij[IntelliJ] are currently available as a tech previews and offer additional in-editor support for your Open Liberty projects. Both extensions contribute commands to easily start and stop Open Liberty dev mode, as well as run tests and view test reports. 

Reviewer: ```suggestion
The Open Liberty Tools extensions for link:https://github.com/OpenLiberty/open-liberty-tools-vscode[VS Code] and link:https://github.com/OpenLiberty/open-liberty-tools-intellij[IntelliJ] are currently available as tech previews and offer additional in-editor support for your Open Liberty projects. Both extensions contribute commands to easily start and stop Open Liberty dev mode, as well as run tests and view test reports. 
```

[#benefits]
=== What benefits do the plugins you mentioned offer?

> IDE plugins, specifically the ones our team works on for MicroProfile and Open Liberty, offer our users an improved experience within their editors. From within their IDE, they are able to use tools like the MicroProfile Starter or Open Liberty dev mode, without having to reference external documentation or visit an external site. This enables developers to focus on the actual code of their application, as opposed to some of the details around initially creating a project, configuring Open Liberty, and deploying to a server. The Liberty developer experience team supports a number of editor plugins, such as the MicroProfile Starter for VS Code, the MicroProfile REST Client Generator for VS Code, and Open Liberty Tools for VS Code and IntelliJ (tech previews). 

Reviewer: ```suggestion
IDE plugins, specifically the ones our team works on for MicroProfile and Open Liberty, offer our users an improved experience within their editors. From within their IDE, they are able to use tools like the MicroProfile Starter or Open Liberty dev mode, without having to reference external documentation or visit an external site. This enables developers to focus on the actual code of their application, as opposed to some of the details around initially creating a project, configuring Open Liberty, and deploying to a server. The Liberty developer experience team supports a number of editor plugins, such as the MicroProfile Starter for VS Code, the Generator for MicroProfile REST Client in VS Code, and Open Liberty Tools for VS Code and IntelliJ. 
```

[#devex]
=== How can open-source projects create better developer experiences? Could you give an example?
By developing our projects in the open-source, we can receive direct feedback from our users through avenues like GitHub issues. Open-source projects also allow us to collaborate with other interested parties in order to build better tools for developers.  For example, both our team and developers at Red Hat are actively contributing to link:https://github.com/eclipse/lsp4mp[Eclipse LSP4MP], the Language Server for MicroProfile. When there is a technology like MicroProfile that is widely used by many developers, it is helpful to be able to leverage the community to build these tools, as they are also the ones that will benefit from using them. 

[#contributions]
=== In addition to contributing to code within these open source projects, what other contributions have you made? (i.e. documentation, tutorials, blogs, etc)

> I have contributed to the documentation on our IDE extensions; Open Liberty Tools for IntelliJ and VS Code, MicroProfile Starter for VS Code, the VS Code Extension Pack for MicroProfile, etc. I have also created blog posts about our Open Liberty Tools for IntelliJ initial tech preview release, and the Language Server for MicroProfile (Eclipse LSP4MP). Beyond these forms of communication, I have spoken at a number of presentations and conferences with the team. 

Reviewer: ```suggestion
I have contributed to the documentation on our IDE extensions; Open Liberty Tools for IntelliJ and VS Code, MicroProfile Starter for VS Code, the VS Code Extension Pack for MicroProfile (Tools for MicroProfile), etc. I have also created blog posts about our Open Liberty Tools for IntelliJ initial tech preview release, and the Language Server for MicroProfile (Eclipse LSP4MP). Beyond these forms of communication, I have spoken at a number of presentations and conferences with the team. 
```
